File: exercises/RB101_small_problems/easy_2/tip_calculator.rb
def tip_calc(total, tip_percent)
  tip_amount = (tip_percent / 100) * total
  puts "The tip is $#{format('%.2f', tip_amount)}\n" + \
       "The total is $#{format('%.2f', total + tip_amount)}"
end

puts "What is the bill?"
bill = gets.chomp.to_f
puts "What is the tip percentage?"
tip = gets.chomp.to_f

tip_calc(bill, tip)

File: exercises/JS210_Small_Problems/Easy_1/tipCalculator.js
const rlSync = require('readline-sync');

console.log('What is the bill? ');
let bill = parseFloat(rlSync.prompt());

console.log('What is the tip percentage? ');
let tip_percentage = parseFloat(rlSync.prompt());

let tip = (tip_percentage / 100) * bill;
let total = tip + bill;

console.log(`The tip is $${tip.toFixed(2)}`);
console.log(`The total is $${total.toFixed(2)}`);